token-metadata/example/Cargo.toml

```diff
@@ -13,15 +13,15 @@ test-sbf = []
 
 [dependencies]
 solana-program = "1.16.13"
-spl-token-2022 = { version = "0.8", path = "../../token/program-2022" }
+spl-token-2022 = { version = "0.9", path = "../../token/program-2022" }
 spl-token-metadata-interface = { version = "0.2.0", path = "../interface" }
 spl-type-length-value = { version = "0.3.0" , path = "../../libraries/type-length-value" }
 spl-pod = { version = "0.1.0", path = "../../libraries/pod" }
 
 [dev-dependencies]
 solana-program-test = "1.16.13"
 solana-sdk = "1.16.13"
-spl-token-client = { version = "0.6", path = "../../token/client" }
+spl-token-client = { version = "0.7", path = "../../token/client" }
 test-case = "3.2"
 
 [lib]
```

Comment:
Do we want to bump the minor version of the Token Metadata example considering these dependency changes?

Reply:
I guess I don't see you doing that in a few other libs, maybe not necessary? Like `stake-pool` and the swap/upgrade/wrap programs.

Reply:
Ok sgtm!